src/App.js

```diff
@@ -1,11 +1,78 @@
-import "./App.css";
+import './App.css';
+
+import { Route, Routes } from 'react-router-dom';
+
+import ClassCompetitorsPage from './PagesBody/ClassCompetitorsPage/ClassCompetitorsPage';
+import ClassesPage from './PagesBody/ClassesPage/ClassesPage';
+import ContactFormPage from './PagesBody/ContactFormPage/ContactFormPage';
+import ContestDetailsPage from './PagesBody/ContestDetailsPage/ContestDetailsPage';
+import ContestsPage from './PagesBody/ContestsPage/ContestsPage';
+import DogDataPage from './PagesBody/DogDataPage/DogDataPage';
+import DogSummaryPage from './PagesBody/DogSummaryPage/DogSummaryPage';
+import ErrorTestPage from './PagesBody/ErrorTestPage/ErrorTestPage';
+import ExercisesPage from './PagesBody/ExercisesPage/ExercisesPage';
+import ForgotPassForm from './Organisms/ForgotPassForm/ForgotPassForm';
+import LandingPage from './PagesBody/LandingPage/LandingPage';
+import Layout from './Templates/Layout/Layout';
+import LeaderboardPage from './PagesBody/LeaderboardPage/LeaderboardPage';
+import LoginForm from './Organisms/LoginForm/LoginForm';
+import ModalsTest from './PagesBody/ModalsTest';
+import NotFoundPage from './PagesBody/NotFoundPage/NotFoundPage';
+import ParticipantDataPage from './PagesBody/ParticipantDataPage/ParticipantDataPage';
+import ProfilePage from './PagesBody/ProfilePage/ProfilePage';
+import RegistrationForm from './Organisms/RegistrationForm/RegistrationForm';
+import RolePage from './PagesBody/RolePage/RolePage';
```

Comment:
Very pretty imports! You can save a lot of conflicts by such a little thing, good job!